consultation_analyser/support_console/jinja2/support_console/ml_pipeline_test.html

```diff
@@ -0,0 +1,61 @@
+{% extends "base.html" %}
+{%- from 'govuk_frontend_jinja/components/button/macro.html' import govukButton -%}
+{%- from 'govuk_frontend_jinja/components/notification-banner/macro.html' import govukNotificationBanner -%}
+{%- from 'govuk_frontend_jinja/components/warning-text/macro.html' import govukWarningText -%}
+
+
+{% block content %}
+
+  {% for message in messages %}
```

Comment:
Let's remove this for now and then we can repave with the #107 version

Reply:
Yup - now removed.